### ISSUE TITLE
zeromq failed to build on 10.13.2 (beta)

### ISSUE THREAD

**Issue by tjrmarques** [Homebrew/homebrew-core#21097]
I'm trying to build zeromq with drafts on 10.13.2 (beta) (Xcode 9.1)
brew install zeromq --with-drafts

Gist is at https://gist.github.com/82d4d9b21448a8b0e7ca7febf2e2c29e

**Comment by ilovezfs** [comment]
```
diff --git a/Formula/zeromq.rb b/Formula/zeromq.rb
index 7d95b5c6e8..462e6a550a 100644
--- a/Formula/zeromq.rb
+++ b/Formula/zeromq.rb
@@ -44,7 +44,8 @@ class Zeromq < Formula
     args << "--enable-drafts" if build.with?("drafts")
 
     ENV["LIBUNWIND_LIBS"] = "-framework System"
-    ENV["LIBUNWIND_CFLAGS"] = "-I#{MacOS.sdk_path}/usr/include"
+    (buildpath/"libunwind_headers").install_symlink MacOS.sdk_path/"usr/include/libunwind.h"
+    ENV["LIBUNWIND_CFLAGS"] = "-I#{buildpath}/libunwind_headers"
 
     system "./autogen.sh" if build.head?
     system "./configure", *args
```

**Comment by ilovezfs** [comment]
or
```
diff --git a/Formula/zeromq.rb b/Formula/zeromq.rb
index 7d95b5c6e8..6f8104e6ef 100644
--- a/Formula/zeromq.rb
+++ b/Formula/zeromq.rb
@@ -44,7 +44,8 @@ class Zeromq < Formula
     args << "--enable-drafts" if build.with?("drafts")
 
     ENV["LIBUNWIND_LIBS"] = "-framework System"
-    ENV["LIBUNWIND_CFLAGS"] = "-I#{MacOS.sdk_path}/usr/include"
+    sdk = MacOS::CLT.installed? ? "" : MacOS.sdk_path
+    ENV["LIBUNWIND_CFLAGS"] = "-I#{sdk}/usr/include"
 
     system "./autogen.sh" if build.head?
     system "./configure", *args
```

**Comment by tjrmarques** [comment]
Built the updated formula from source, it works.
Thanks!

**Comment by ilovezfs** [comment]
Thanks for reporting this. I've merged #21100 now.
